src/templates/index.html

```diff
@@ -5,7 +5,7 @@
 <html>
 
         <title><%=settings.title%></title>
-    <script>
+    <script nonce="<%=settings.nonce && settings.nonce.padLicense || ''%>">
```

Comment:
If `settings.nonce.padLicense` is nullish/empty, wouldn't it be better to not add the `nonce` attribute? Like this:

```suggestion
    <script <%=(settings.nonce && settings.nonce.padLicense) ? `nonce="${settings.nonce.padLicense}"` : ''%>>
```